testutil/testfactory/types.go

```diff
@@ -0,0 +1,62 @@
+package testfactory
```

Comment:
[optional] since this file doesn't define any new types, proposal to rename it from `types.go` to something else?

Proposals:
1. `generate.go`
2. `generate_tx.go`

Reply:
Thanks for explaining the rationale. Given that rationale, I'm comfortable with the name: `testutil/testfactory/types.go`. I'm also comfortable with splitting the generation into `generate_tx.go` and `generate_blob.go` so defer to you.

Reply:
Cool, changing to `generate_txs.go` and `generate_blob.go` for now, will also wait for @evan-forbes if he have any input here since I saw a similar structure being used in `celestia-core` (https://github.com/celestiaorg/celestia-core/blob/v0.34.x-celestia/test/factory/tx.go#L1)